src/DBmysql.php

```diff
@@ -363,10 +363,9 @@ public function query($query)
             $SQL_TOTAL_REQUEST++;
             $DEBUG_SQL["queries"][$SQL_TOTAL_REQUEST] = $query;
         }
-        if ($is_debug || $this->execution_time === true) {
-            $TIMER = new Timer();
-            $TIMER->start();
-        }
+
```

Comment:
I do not really understand why this is no longer under a condition. Just for `$TIMER` to be defined?

Reply:
(Looking at `Timer` class, this should not really impact)

Reply:
> I do not really understand why this is no longer under a condition. Just for `$TIMER` to be defined?

PHPStan is not able to recognize that `$TIMER` is always defined where it is used.